client/components/tinymce/plugins/wpcom/plugin.js

```diff
@@ -289,10 +289,15 @@ function wpcomPlugin( editor ) {
 
 	// Remove spaces from empty paragraphs.
 	editor.on( 'BeforeSetContent', function( event ) {
-		var paragraph = tinymce.Env.webkit ? '<p><br /></p>' : '<p></p>';
-
 		if ( event.content ) {
-			event.content = event.content.replace( /<p>(?:&nbsp;|\u00a0|\uFEFF|\s)+<\/p>/gi, paragraph );
+			var paragraph = tinymce.Env.webkit ? '<p><br /></p>' : '<p></p>';
```

Comment:
This is an interesting move because as a `var` it still gets hoisted up to where it was originally defined. @timmyc do you think it would be worth recreating this as a `const` given that it has been moved inside the `if` block?

(hint: according to our style guide we don't declare `var` bindings below the top of a function definition for this very reason)